### ISSUE TITLE
is ignoring objects counter-intuitive or sensible?

### ISSUE THREAD

**Issue by evanwill** [CollectionBuilder/collectionbuilder-csv#26]
@dcnb @owikle I think 6b8f3102e1531c5cdd1ef31d332ebd1a7904bd63 should be reverted? This isn't GH, and the idea is larger collections where the objects are hosted somewhere else. The object dir has been set up sort of like a temp work space to use with the rake task (i.e. in practice I usually do the rake processing, test things locally using the base urls generated in the "object_list.csv" with the local copies, then move the objects to a web hosting location and update the metadata urls with the appropriate prefix). In general, you shouldn't be putting a big collection of binary files onto github--which new users don't necessarily get. The docs try to explain the workflow using the rake task and moving the files--but it would be easy to accidentally commit all the files you were just testing with. 

So I think objects/ should stay (i.e. go back to) ignored

**Comment by dcnb** [comment]
@evanwill I can be convinced, and you've got me some of the way, but I'd like to talk it out a bit in sprint tomorrow. I feel like this is somewhat an issue between modeling "correct" behavior vs. not standing in the way of the user. I've built big collections with objects ignored and I've also unignored it with decent size collections. I changed it after working on the new cdil site last week because I was frustrated that all the image/rake work I did did not follow me home. Let's talk about this tomorrow tho and then decide if we want to revert it. Thanks for bringing it up.
